packages/terra-hyperlink/src/Hyperlink.jsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Hyperlink.scss';
+
+const cx = classNames.bind(styles);
+
+const KEYCODES = {
+  ENTER: 13,
+  SPACE: 32,
+  TAB: 9,
+};
+
+const variants = {
+  DEFAULT: 'default',
+  EXTERNAL: 'external',
+  IMAGE: 'image',
+  VIDEO: 'video',
+  AUDIO: 'audio',
+  DOCUMENT: 'document',
+};
+
+const propTypes = {
+  /**
+   * The content to display inside link.
+   */
+  children: PropTypes.node,
+  /**
+   * Sets the href of the link.
+   */
+  href: PropTypes.string,
+  /**
+   * Whether or not the link should be disabled.
+   */
+  isDisabled: PropTypes.bool,
+  /**
+   * Whether or not the link should display and underline by default.
```

Comment:
> Whether or not the link should display ~and~ **an** underline ~by default~.

Reply:
We should indicate the underline style is still applied on hover.

Reply:
Updated: https://github.com/cerner/terra-core/pull/1562/commits/5d1a6de0b4dd80fbbb620354ee5b823764386a5b

Reply:
Updated again, missed another typo: https://github.com/cerner/terra-core/pull/1562#discussion_r193008539